pyproject.toml

```diff
@@ -1,6 +1,6 @@
 # pyproject.toml
 [build-system]
-requires = ["setuptools>=45", "wheel", "setuptools_scm[toml]>=6.0"]
+requires = ["setuptools>=45", "setuptools_scm[toml]>=6.0"]
```

Comment:
I seem to recall `"setuptools>=61"` is requited for PEP517 compliance. Is that why you explicitly require  "wheel"?